nntrainer/layers/acti_func.h

```diff
@@ -588,6 +618,8 @@ class ActiFunc {
   }
 
 private:
+  constexpr static inline float ALPHA = 1.0f; /**< alpha for elu */
```

Comment:
quick question! any reason to set `ALPHA` as private while `NEGATIVE_SLOPE` is public?

Reply:
This value is used in only internal implementation. So, I think it's better to hide `ALPHA` from public API. In my opinion, `NEGATIVE_SLOPE` should be private too but I thought it'll be better to make a separated PR for it.